Src/Microsoft.Scripting/Microsoft.Scripting.csproj

```diff
@@ -1,8 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
You're doing the separate target/props import instead of using sdk in the project tag so that the BeforeBuild/AfterBuild targets work? Perhaps a comment to that effect in the project files would be useful.